src/main/java/UI/UserLikeBlock.java

```diff
@@ -167,29 +174,37 @@ public UserLikeBlock(UserAccount user1, UserAccount user2){
     public void actionPerformed(ActionEvent e) {
         // if like button get clicked
         if (e.getSource()== like) {
-            String textShow= controller.likeController(user1, user2);
+            String textShow= controller.likeController(username1, username2);
             likedString.setText(textShow);
             likedString.setVisible(true);
         }
         //if blocked button get clicked
         else if (e.getSource()==block) {
-            String textShow2= controller.blockController(user1, user2);
+            String textShow2= controller.blockController(username1, username2);
             blockedString.setText(textShow2);
             blockedString.setVisible(true);
         }
     }
 
     public void functionToCall(){
-        new UserLikeBlock(user1, user2);
+        new UserLikeBlock(username1, username2);
     }
 
     public static void main (String[] args) {
 
+//        LoginController loginController= new LoginController();
```

Comment:
You might want to leave an annotation that this is code for your own testing (but that it is not a unit test).